.travis.yml

```diff
@@ -21,7 +21,7 @@ deploy:
   provider: releases
   # `token` is an encrypted oauth token for rafecolton. See https://docs.travis-ci.com/user/deployment/releases
```

Comment:
Does this comment need to be updated now?

Reply:
Updated!